### PR TITLE
Change go redis version to v8.4.9 in apmgoredisv8 and update the test

### PR DESCRIPTION
**Pull request by sangianpatrick**
ClusterClient wraps the command with MULTI/EXEC in go redis v8.4.9

### CODE REVIEW

File: module/apmgoredisv8/hook_test.go [hunk 1]
```diff
@@ -120,14 +120,14 @@ func TestHookTxPipeline(t *testing.T) {
...
-			if _, ok := client.(*redis.Client); ok {
-				// *redis.Client wraps queued commands with MULTI/EXEC
-				// in (*redis.Client) processTxPipeline
-				assert.Equal(t, "MULTI, GET, SET, GET, (empty command), EXEC", spans[0].Name)
-			} else {
-				// *redis.ClusterClient and *redis.Ring don't wrap queued commands
-				// with MULTI/EXEC in (*redis.Client) processTxPipeline
+			if _, ok := client.(*redis.Ring); ok {
+				// *redis.Client wraps queued commands without MULTI/EXEC
+				// in (*redis.Ring) processTxPipeline
```
Comment (axw) [inline module/apmgoredisv8/hook_test.go:125]:
```suggestion
				// *redis.Ring does not wrap queued commands with MULTI/EXEC
				// in (*redis.Ring) processTxPipeline
```

I think? IIUC the only change is that ClusterClient now *does* wrap them.

**Review by axw — APPROVED** [review]
LGTM, thanks @sangianpatrick! I think there's a minor adjustment needed in a comment as noted. Please take a look and update, and then I'll merge.

**Comment by axw** [comment]
jenkins run the tests please

File: module/apmgoredisv8/hook_test.go (hunk 1 above)
Comment (sangianpatrick) [reply module/apmgoredisv8/hook_test.go]: @axw thanks for reminding me about the comment, already fixed it.

**Comment by axw** [comment]
@sangianpatrick the tests are failing with Go 1.11 and 1.12:

> [2021-01-25T04:13:39.534Z] ../../../../../pkg/mod/github.com/go-redis/redis/v8@v8.4.9/redis.go:222:13: undefined: errors.Unwrap

go-redis has bumped the minimum required version of Go to 1.13 in a minor release. If we were to increase the version in the agent, then it would break users who are still on older versions of Go. We consider that a breaking change, and not something we will do in a minor version.

I think this change would need to go into https://github.com/elastic/apm-agent-go/issues/654, or otherwise changes are required in go-redis to support older versions of Go.

You can still bump the version of go-redis in your application's `go.mod` and the instrumentation should work as expected.

**Comment by axw** [comment]
Closing - can't update because of the non-semver change upstream. Thanks all the same.

**Pull request closed without merging**